ultet/src/main/scala/za/co/absa/ultet/util/CliParser.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.ultet.util
+
+import scopt.OParser
+
+case class Config(yamlSource: String = "",
```

Comment:
IntelliJ is complaining about this:

`Class doesn't correspond to file name `

perhaps it can be extracted (and renamed?) into its own file, e.g. `AppConfig`

Reply:
Fixed